src/profile-logic/tracks.js

```diff
@@ -266,6 +393,30 @@ export function computeLocalTracksByPid(
       // This thread has IPC markers.
       tracks.push({ type: 'ipc', threadIndex });
     }
+
+    const timelineAwareMarkerConsts = new Set(
+      timelineAwareMarkers.map((marker) =>
+        thread.stringTable.indexForString(marker.name)
+      )
+    );
+    const containedMarkerConsts = new Set(
+      thread.markers.name.filter((x) => timelineAwareMarkerConsts.has(x))
```

Comment:
I can't think properly right now at that time of the day, but it seems to me we can do better than iterating over all markers. Especially we should be able to stop looping as soon as we find one marker that matches the marker schema we're targeting.